warehouse/query-core/src/main/java/datawave/query/jexl/functions/QueryFunctionsDescriptor.java

```diff
@@ -190,9 +191,14 @@ private static void verify(String name, int numArgs) {
                 }
                 break;
             case QueryFunctions.UNIQUE_FUNCTION:
-            case QueryFunctions.UNIQUE_BY_DAY_FUNCTION:
-            case QueryFunctions.UNIQUE_BY_HOUR_FUNCTION:
-            case QueryFunctions.UNIQUE_BY_MINUTE_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_DAY_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_YEAR_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_MINUTE_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_HOUR_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_MONTH_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_TENTH_OF_HOUR_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_SECOND_FUNCTION:
+            case QueryOptionsFromQueryVisitor.UniqueFunction.UNIQUE_BY_MILLISECOND_FUNCTION:
```

Comment:
Same comment with regard to sorting either alphabetically or by logical time unit. Not required for an approval.